content/fr/docs/setup/_index.md

```diff
@@ -0,0 +1,93 @@
+---
+reviewers:
+- brendandburns
+- erictune
+- mikedanese
+no_issue: true
+title: Setup
+main_menu: true
+weight: 30
+content_template: templates/concept
+---
+{{% capture overview %}}
+
+Utilisez cette page pour trouver le type de solution qui correspond le mieux à vos besoins.
+
+Le choix de l'emplacement de Kubernetes dépend des ressources dont vous disposez 
+et de la flexibilité dont vous avez besoin. Vous pouvez executer Kubernetes presque partout, 
+de votre ordinateur portable aux machines virtuelles d'un fournisseur de cloud jusqu'à un rack de serveurs en bare metal.
+Vous pouvez également mettre en place un cluster entièrement géré en exécutant une seule commande ou bien créer 
+votre propre cluster personnalisé sur vos serveurs bare-metal.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Solutions pour machine locale
```

Comment:
Suggest:  [...] `pour une machine locale` or `Solutions locales`

Reply:
+1 for "Solutions locales", "machines locale" sounds weird

Reply:
totally agree, solution locale is better

Reply:
pour préciser, je propose: La solution locale, installée sur votre machine, est un moyen fa...